front/lib/labs/transcripts/utils/helpers.ts

```diff
@@ -32,16 +32,17 @@ export async function getGoogleAuthFromUserTranscriptsConfiguration(
   auth: Authenticator,
   userId: ModelId
 ) {
-  const providerId = "google_drive";
-
   const transcriptsConfiguration =
-    await LabsTranscriptsConfigurationResource.findByUserWorkspaceAndProvider({
+    await LabsTranscriptsConfigurationResource.findByUserWorkspace({
```

Comment:
We likely want to fix the unique index on  `LabsTranscriptsConfigurationModel` ?

Reply:
Good catch. Changed to `{ fields: ["userId", "workspaceId"], unique: true },`